com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/HDShaderGUI.cs

```diff
@@ -106,7 +106,7 @@ protected static void SynchronizeShaderGraphProperties(Material material)
         {
             var defaultProperties = new Material(material.shader);
             foreach (var floatToSync in floatPropertiesToSynchronize)
-                if (material.HasProperty(floatToSync))
+                if (material.HasProperty(floatToSync) && defaultProperties.HasProperty(floatToSync))
```

Comment:
Indeed I do remember specifically hitting an issue with this line originally failing to catch a non-existent default property (I did not investigate too much into the why) for the Unlit shader (`RequiresSplitLighting` I believe). So, I added this case to the predicate to protect against the case of a non-existent default property, and haven't encountered the issue since (though it seemed to happen randomly and was hard to reproduce).